sw/device/tests/crypto/BUILD

```diff
@@ -124,7 +124,7 @@ opentitan_functest(
         "dv",
     ],
     verilator = verilator_params(
-        timeout = "long",
+        timeout = "eternal",
```

Comment:
This wont be enough, even with https://github.com/lowRISC/opentitan/pull/16782

I don't even know if this has trouble with vector because it doesn't make it that far.